.github/workflows/stale.yml

```diff
@@ -0,0 +1,23 @@
+name: Stale
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: harupy/stale@mlflow-stale-bot
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 7 days with no activity. Remove stale label or comment or this will be closed in 14 days.'
+          close-issue-message: 'This issue was closed because it has been stalled for 14 days with no activity.'
+          days-before-stale: 7
+          days-before-close: 14
```

Comment:
Can we do `days-before-stale: 14` and `days-before-close: 28`? Many folks take 2-week vacations, and we wouldn't want their issues to be closed during that time.

Reply:
Yes, we can :)

Reply:
Updated!